news/2021-12-17-winter-fanart-contest-results.md

```diff
@@ -0,0 +1,123 @@
+
+Fanart contest regular [**Dem4eg-**](https://osu.ppy.sh/users/7206818) won over **4836** voters with a timid pippi offering a festive tithe of osu!supporter and a pen tablet all neatly adorned with a ribbon.
+
+Simple, clean, and a frontrunner in the voting from the very start of the contest, Dem4eg- joins the lineup of hallowed fanart finalists for the first time with a real stonker of an entry, no less. Congratulations!
+
+[![](https://assets.ppy.sh/contests/136/banners/lutfi_anims.jpg)](https://assets.ppy.sh/contests/136/winners/lutfi_anims.png)
+
+**4685** approved of [**lutfi_anims**](https://osu.ppy.sh/users/11041959)'s depiction of pippi and Mocha catching a moment in the snow beneath a bare winter tree. Facial expression takes front and center in this one, showing you what the community's really looking for when they vote!
+
+[![](https://assets.ppy.sh/contests/136/banners/eulorie.jpg)](https://assets.ppy.sh/contests/136/winners/eulorie.png)
+
+[**eulorie**](https://osu.ppy.sh/users/20349380)'s winter wonderland enchanted **4500** voters with promises of smores and adorable outfits in equal measure, toting a festively handsome pippi and a most adorable Chirou with a Mocha backpack to boot!
+
+The overall vibes of this artwork are really something else, and hearken back to the days of old MMORPG festive loading screens like Ragnarok Online, if anybody reading these newsposts is even of the age to remember these games any more. What a piece!
+
+[![](https://assets.ppy.sh/contests/136/banners/Tin_Can.jpg)](https://assets.ppy.sh/contests/136/winners/Tin%20Can.png)
+
+[**Tin Can**](https://osu.ppy.sh/users/10682807)'s take on a smiling pippi warming herself up with an osu!-brand hot drink earned the approval of **4497** voters, all while Mocha and a don take to ice skating off elsewhere in shot.
+
+[![](https://assets.ppy.sh/contests/136/banners/Dreamxiety.jpg)](https://assets.ppy.sh/contests/136/winners/Dreamxiety.png)
+
+[**Dreamxiety**](https://osu.ppy.sh/users/13103233) is back at it again with **3982** voters thoroughly impressed by a doge-toting pippi enjoying a moment whilst Aiko and Chirou build a snowman in a rather scenic wintry vista. Simply superb, as always!
+
+[![](https://assets.ppy.sh/contests/136/banners/handakote.jpg)](https://assets.ppy.sh/contests/136/winners/handakote.png)
+
+[**handakote**](https://osu.ppy.sh/users/23188287)'s undeniable style snared **3839** voters with Mocha and pippi both toting very stylish winter outfits with a splash of floral accent. It's all in the eyes, baby!
+
+[![](https://assets.ppy.sh/contests/136/banners/ShovelKun.jpg)](https://assets.ppy.sh/contests/136/winners/ShovelKun.png)
+
+**3765** voters were impressed by [**ShovelKun**](https://osu.ppy.sh/users/9948284)'s festive urban delve into an aloof pippi and unmistakably cute Mocha posing for the camera before a muted cityscape. 
```

Comment:
syntax error

```suggestion
**3765** voters were impressed by [**ShovelKun**](https://osu.ppy.sh/users/9948284)'s festive urban take of an aloof pippi and unmistakably cute Mocha posing for the camera before a muted cityscape. 
```

Reply:
delve into technically correct here and just a wording thing on my part, i'm reluctant to use 'take of' any further since there's a *lot* of it already

Reply:
so you're treating delve in "delve into" as a noun? there is an unrelated archaic definition for delve as a noun, so i still think your usage here is sus

![D87943E4-0D59-4398-B36D-91AA682927EA](https://user-images.githubusercontent.com/36758269/146540956-4c9cbe18-ce7f-4d86-b915-51fd85b457d0.jpeg)

Reply:
perhaps like this

```suggestion
**3765** voters were impressed to see [**ShovelKun**](https://osu.ppy.sh/users/9948284) delve into a festive urban style with an aloof pippi and unmistakably cute Mocha posing for the camera before a muted cityscape. 
```

Reply:
hmm upon further inspection it seems you can treat it as a noun like that, but then it still doesn't quite make sense as-is ("a delve into an aloof pippi and co.")

Reply:
delving into an aloof pippi sounds ... perverted?